src/reduxStore/todoSlice.js

```diff
@@ -12,12 +12,21 @@ export const todoSlice = createSlice({
   initialState,
   reducers: {
     addTodo: (state, action) => {
-      state.todos.unshift({ ...action.payload, status: "active" });
+      const date = new Date();
+      const year = date.getFullYear();
+      const month = date.getMonth()+ 1;
+      const day = date.getDate() 
+
+      const dateCreated = `${day}/${month}/${year}`;
```

Comment:
rename `dateCreated` to `created_at`

Reply:
also add `completed_at`

Reply:
<img width="308" alt="image" src="https://github.com/Highshirt99/TODOTODAY/assets/53216981/23864d71-229d-4e6d-9832-192cd9bbae56">